packages/components/src/BrandMoment/BrandMoment.module.css

```
@@ -160,20 +162,22 @@ $breakpoint-xl: 1366px; // TODO: create an xl breakpoint in design-tokens and re
   }
 }
 
+/*
 // --------------------------------
 // Footer
 // --------------------------------
+*/
```

I suggest you just simplify these to a smaller block, or at least make the comment symbols match haha. Or even delete them. They actually aren't required.

```suggestion
/*
 * Footer
 */
```

```suggestion
/* Footer */
```